ee/clickhouse/queries/trends/trend_event_query.py

```diff
@@ -31,6 +31,17 @@ def get_query(self) -> Tuple[str, Dict[str, Any]]:
                 )
             )
             + (f", {self.DISTINCT_ID_TABLE_ALIAS}.person_id as person_id" if self._should_join_distinct_ids else "")
+            + (
+                " ".join(
+                    f", {self.EVENT_TABLE_ALIAS}.{column_name} as {column_name}" for column_name in self._extra_fields
```

Comment:
I gather we don't need to do argMax here because there will only be one vary for each event_id, column?

Reply:
We don't do a group by here. In persons table we need to select which value to use due to group by :)